Example/Tests/ConfigurationTests/CLDAnalyticsTests.swift

```diff
@@ -12,11 +12,17 @@ import XCTest
 class CLDAnalyticsTests: BaseTestCase {
 
     func test_analyicsString() {
-        var analyticsString = CLDAnalytics.shared.generateAnalyticsSignature(sdkVersionString: "1.24.0",techVersionString: "12.0")
-        XCTAssertEqual(analyticsString, "CAEAlhAMBAM0")
+        var analyticsString = CLDAnalytics().generateAnalyticsSignature(sdkVersionString: "1.24.0",techVersionString: "12.0")
+        XCTAssertEqual(analyticsString, "DAEAlhAMBMA0")
 
-        analyticsString = CLDAnalytics.shared.generateAnalyticsSignature(sdkVersionString: "1.24.0-beta.6",techVersionString: "12.0")
-        XCTAssertEqual(analyticsString, "CAEAlhAMBAM0")
+        analyticsString = CLDAnalytics().generateAnalyticsSignature(sdkVersionString: "1.24.0-beta.6",techVersionString: "12.0")
+        XCTAssertEqual(analyticsString, "DAEAlhAMBMA0")
+
+        analyticsString = CLDAnalytics().generateAnalyticsSignature(sdkVersionString: "1.24.0",techVersionString: "16.3")
+        XCTAssertEqual(analyticsString, "DAEAlhE8BQD0")
+
+        analyticsString = CLDAnalytics().generateAnalyticsSignature(sdkVersionString: "1.24.0",techVersionString: "17.1")
+        XCTAssertEqual(analyticsString, "DAEAlhB1BRB0")
```

Comment:
techVersion is not OS version. Where are the tests for OS version?

Reply:
The `techVersionString` is the iOS version, we send it twice